ibis/backends/tests/test_examples.py

```diff
@@ -49,40 +49,3 @@ def test_load_examples(con, example, columns):
     t = getattr(ibis.examples, example).fetch(backend=con)
     assert t.columns == columns
     assert t.count().execute() > 0
-
-
-@pytest.mark.skipif(
-    (LINUX or MACOS) and SANDBOXED,
-    reason="nix on linux cannot download duckdb extensions or data due to sandboxing",
-)
-@pytest.mark.notimpl(
-    [
-        # everything except duckdb
-        "bigquery",
-        "clickhouse",
-        "dask",
-        "datafusion",
-        "druid",
-        "exasol",
-        "flink",
-        "impala",
-        "mssql",
-        "mysql",
-        "oracle",
-        "pandas",
-        "polars",
-        "postgres",
-        "risingwave",
-        "pyspark",
-        "snowflake",
-        "sqlite",
-        "trino",
-    ]
-)
-def test_load_geo_example(con):
```

Comment:
Seemed pointless to keep this as a backend test since it only works with duckdb right now.

Reply:
I moved it to `ibis/backends/duckdb/tests/test_geospatial.py`